SchoolManagerSystem.Common/Enums/UserRole.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SchoolManagerSystem.Common.Enums
+{
+	public enum UserRole
+	{
+		Principal,
```

Comment:
too many indentation

Reply:
i have cleared everything and the spaces between props